resources/views/livewire/cooperation/frontend/tool/simple-scan/my-regulations.blade.php

```diff
@@ -0,0 +1,104 @@
+<div>
+    <div class="w-full flex flex-wrap" x-data="{ selected: null }">
+
+        <div class="w-full flex flex-wrap justify-between" x-data="tabs(@if(request()->has('tab')) '{{ request()->get('tab') }}' @endif)">
+            <nav class="nav-tabs" x-show="selected === null">
+                @foreach(__('cooperation/frontend/tool.my-regulations.categories') as $key => $category)
+                    <a x-bind="tab" href="#" @if($loop->first) x-ref="main-tab" @endif data-tab="{{ $key }}">
+                        {{ str_replace(':count', count(($relevantRegulations[$key] ?? [])), $category) }}
+                    </a>
+                @endforeach
+            </nav>
+            <div class="flex items-center gap-x-2" wire:poll="checkIfIsRefreshed">
```

Comment:
`@if($isRefreshing) wire:poll="checkIfIsRefreshed" @endif`, just like in `download-pdf.blade.php`, to keep requests to a minimum when not needed